File: Java/FJ - 11/4.14 Programa 1/src/Pessoa.java

public class Pessoa {
	String nome;
	int idade;
	
	public void fazAniversario() {
		idade++;
	}
	
	public void mostraNomeEIdade() {
		System.out.println(String.format("Nome: %s\nIdade: %d", nome, idade));
	}
}

File: C#/Treinamento GFT/Aula 08 - Prova Pessoas/Aula 08 - Prova Pessoas/Pessoa.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Aula_08___Prova_Pessoas
{
    class Pessoa
    {
        public Pessoa(string nome, int idade)
        {
            Nome = nome;
            Idade = idade;
        }

        public string Nome { get; }
        public int Idade { get; }
    }
}
